src/components/itineraries/Generate.tsx

```diff
@@ -80,6 +84,9 @@ const Generate: FC = () => {
   const showWeekDay = isWeekDay()
   const showWeekend = !showWeekDay
 
+  const { addAddress } = parseLocationSearch(window.location.search)
+  const showAddAddressModal = addAddress === '1'
```

Comment:
Maybe more aesthetic to use hash here?

Reply:
Yeah, I agree.
Was already doubting this on implementing.

I'll refactor to use a hash here.